### DISCUSSION TITLE
Evo 16x16 and Pixoo 32x32

### DISCUSSION THREAD

**Discussion by KC-inDomus** [mathoudebine/homeassistant-timebox-mini#7] [Q&A]
Hi there,
is your component expected to work with the other two models Evo and Pixoo too?

**Comment by mathoudebine** [comment]
Hello,

The component could work with these 2 devices if they follow the same communication protocol than the Mini, which should be the case.
You can try it as-is and see if the commands are working (clock display, picture/animations...)

If yes, then you will have to:
- create 16x16 or 32x32 PNG/GIF for your devices and put them in `custom_components/timebox_mini/matrices` for PNG / `custom_components/timebox_mini/animations` for GIF, then list them in the [services.yaml](https://github.com/mathoudebine/homeassistant-timebox-mini/blob/main/custom_components/timebox_mini/services.yaml)
- edit the size for pictures/animations that is actually hard-coded to 11
https://github.com/mathoudebine/homeassistant-timebox-mini/blob/54e7fb6807f7d15ecdda71fdcb5f24873014ada1/custom_components/timebox_mini/__init__.py#L211
https://github.com/mathoudebine/homeassistant-timebox-mini/blob/54e7fb6807f7d15ecdda71fdcb5f24873014ada1/custom_components/timebox_mini/__init__.py#L216

Let me know if this works for you
